scripts/push-stateless-js-idls.sh

```diff
@@ -5,7 +5,12 @@ SOURCE_DIR="./target/types"
 DEST_DIR="./js"
 DEST_DIR_STATELESS="$DEST_DIR/stateless.js/src/idls"
 
-FILES_TO_COPY=("account_compression.ts" "light.ts" "psp_compressed_pda.ts" "user_registry.ts")
+FILES_TO_COPY=(
+  "light_account_compression.ts"    
```

Comment:
`light_account_compression` -> `account_compression` after all i think is the current consensus